reasoner/controller/ConcludableController.java

```diff
@@ -248,9 +258,27 @@ protected void mayAddTraversal() {
             private FunctionalIterator<ConceptMap> filterInferred(ConceptMap conceptMap) {
                 // TODO: Requires duplicate effort to filter out inferred concludable answers. Instead create a new
                 //  traversal mode that doesn't return inferred concepts
-                if (concludable.isInferredAnswer(conceptMap)) return Iterators.empty();
+                if (concludable.isInferredAnswer(conceptMap)) return empty();
                 for (Concept c : conceptMap.concepts().values()) {
-                    if (c.isThing() && c.asThing().isInferred()) return Iterators.empty();
+                    if (c.isThing() && c.asThing().isInferred()) return empty();
+                }
+                return Iterators.single(conceptMap);
+            }
+
+            @Override
+            protected FunctionalIterator<ConceptMap> rejectMismatchedInference(ConceptMap conceptMap) {
+                Concept conceptToCheck = null;
+                if (concludable.isAttribute()) {
+                    conceptToCheck = conceptMap.get(concludable.asAttribute().attribute().id());
+                } else if (concludable.isIsa()) {
+                    conceptToCheck = conceptMap.get(concludable.asIsa().isa().owner().id());
+                }
+                if (conceptToCheck != null) {
+                    if (conceptToCheck.asThing().isInferred()) {
+                        return Iterators.single(conceptMap);
+                    } else {
+                        return empty();
+                    }
```

Comment:
This is coded to specifically catch the scenario where: 
1. the rule conclusion is a `has explicit`
2. the attribute concept is actually an inserted fact
3. the concludable only concerns the attribute and not the `has` edge, in which case this answer no longer represents an inferred answer to the concludable

Not rejecting this answer at this stage leads to an extra explanation, which we deem invalid according to our definition of what should be explained (only inferences should be explained).